packages/contracts-bedrock/src/libraries/Types.sol

```diff
@@ -75,4 +75,22 @@ library Types {
         L1,
         L2
     }
+
+    /// @notice Enum representing different types of configurations that can be set on L1BlockInterop.
+    /// @custom:value GAS_PAYING_TOKEN  Represents the config type for setting the gas paying token.
+    /// @custom:value BASE_FEE_VAULT_CONFIG Represents the config type for setting the base fee vault configuration.
+    /// @custom:value L1_FEE_VAULT_CONFIG  Represents the config type for setting the L1 fee vault configuration.
+    /// @custom:value SEQUENCER_FEE_VAULT_CONFIG Represents the config type for setting the sequencer fee vault
+    ///               configuration.
+    /// @custom:value ADD_DEPENDENCY        Represents the config type for adding a chain to the interop dependency set.
+    /// @custom:value REMOVE_DEPENDENCY     Represents the config type for removing a chain from the interop dependency
+    /// set.
+    enum ConfigType {
+        GAS_PAYING_TOKEN,
+        BASE_FEE_VAULT_CONFIG,
+        L1_FEE_VAULT_CONFIG,
+        SEQUENCER_FEE_VAULT_CONFIG,
+        ADD_DEPENDENCY,
```

Comment:
Im a bit scared of adding new enums in the middle of an already existing one since we dont know if we could be breaking any integration with it since enums are internally represented as sequential integers starting from 0. This means that if you have an integer value, you can cast it to the corresponding enum type to "access" the enum value at that index.
We should probably notify to the devx team about this when the PR is ready